.cirrus.yml

```
@@ -6,6 +6,18 @@ env:
   CHANNEL: "master" # Default to master when not explicitly set by a task.
   PLUGIN_TOOL_COMMAND: "dart ./script/tool/bin/flutter_plugin_tools.dart"
 
+install_chrome_linux_template: &INSTALL_CHROME_LINUX
+  env:
+    CHROME_NO_SANDBOX: true
+    CHROME_DOWNLOAD_DIR: /tmp/chromium
+    CHROME_EXECUTABLE: $CHROME_DOWNLOAD_DIR/chrome-linux/chrome
+    CHROMEDRIVER_EXECUTABLE: $CHROME_DOWNLOAD_DIR/chromedriver/chromedriver
+    PATH: $PATH:$CHROME_DOWNLOAD_DIR/chrome-linux
+  install_chromium_script:
+    # Install a pinned version of Chromium and its corresponding ChromeDriver.
+    # Setting CHROME_EXECUTABLE above causes this version to be used for tests.
+    - ./script/install_chromium.sh "$CHROME_DIR"
```

```suggestion
    - ./script/install_chromium.sh
```

I just realized that `install_chromium.sh` doesn't look at `$1`, so we could get rid of it from the call.

At first I thought this was a bug I had introduced in flutter/packages and that it worked because `CHROME_DIR` is undefined so we'd never put the wrong value in `$1`, but it works because we only look at `CHROME_DOWNLOAD_DIR` in the `.sh`, and it defaults to the same value as the `.cirrus.yml` version (`/tmp/chromium`)

---

Oops, I thought I did that since I noticed that as well.